app/views/layouts/course_settings.html.slim

```diff
@@ -0,0 +1,15 @@
+= render within_layout: controller.parent_layout(of_layout: 'course_settings') do
+  ul.nav.nav-tabs role="tablist"
+    - if params[:controller] == 'course/course_settings'
```

Comment:
still don't get you. can you attach a screenshot?

i don't think the conditional is the right way to do it here. Courses can also register themselves as callbacks in the list of registered modules.